tests/test_nxviz.py

```diff
@@ -114,3 +137,20 @@ def test_init_node_colors():
     G = make_graph_for_grouping()  # noqa
     b = BasePlot(graph=G)
     assert len(set(b.node_colors)) == 1
+
+
+def test_init_edge_colors():
+    """
+    Does two checks:
+    1. If edge_color is not passed in as a keyword argument, check that
+       self.edge_colors is a list of 'blue', of length (number of nodes).
```

Comment:
I think the code in `plots.py` states that it should be a list of 'black'. Please change this accordingly.